app/Product.php

```diff
@@ -9,20 +9,35 @@ class Product extends Model
 {
     use DataViewer;
 
-    protected $fillable = ['name', 'feature', 'user_id', 'brand_id', 'category_id', 'retail_price'];
+    protected $fillable = ['name', 'feature', 'user_id', 'brand_id', 'category_id', 'retail_price',  'is_active', 'img_url'];
 
-    public static $columns = ['id', 'name', 'retail_price in Naira'];
+    public static $columns = ['id', 'name', 'retail_price in Naira', 'brand', 'category', 'status'];
 
     public static function form(): iterable
```

Comment:
why are we having this form function on all models?

Reply:
It is an obvious repetition. We need to refactor it. I had to continue that way cos i was new on the Job